src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -512,6 +521,66 @@ The trusted code can store capabilities on the trusted stack when calling out of
 when returning to the same compartment.
 =====
 
+[#ycfg,reftext="ycfg"]
+===== CHERI Capability Encoding (__x__ycfg)
+
+The <<ycfg>> register is used to identify which CHERI capability encoding is used by the platform.
+The capability encoding both determines the in-memory representation of a CHERI capability
+and entails a set of extensions present atop {cheri_base_ext_name}.
+The CSR exists in the _writable_ namespace and all bits are defined to be WARL,
+but, absent any future extensions, it is expected that each implementation has exactly one legal value.
+This CSR is always present in the machine mode read/write CSR space;
+if the hart supports user mode, a `uycfg` register exists,
+if the hart supports supervisor mode, an `sycfg` register exists, and
+if the hart supports hypervisor mode, a `vsycfg` register exists.
+`sycfg` is always read-only, as is `vsycfg`, if it exists.
+`uycfg` is a read-only alias of `sycfg`, if it exists, or of `mycfg` otherwise.
+If the hypervisor extension is enabled, then `sycfg` is WARL,
+updatable from M-mode or HS-mode with <<pcc>> granting <<asr_perm>>.
+
+.CHERI capability encoding CSR (`__x__ycfg`) format
+[#ycfg_csr_structure]
+[bytefield]
+----
+(defattrs :plain [:plain {:font-family "M+ 1p Fallback" :font-size 24}])
+(def row-height 40)
+(def row-header-fn nil)
+(def left-margin 100)
+(def right-margin 100)
+(def boxes-per-row 32)
+(draw-column-headers {:height 20 :font-size 18 :labels (reverse (concat ["0"] (repeat 6 "") ["8" "9"] (repeat 22 "") ["XLEN-1"]))})
+
+(draw-box "Features" {:span 24})
+(draw-box "Base Encoding" {:span 8})
+
+(draw-box "MXLEN-8"  {:span 24 :borders {}})
+(draw-box "8"        {:span 8 :borders {}})
+----
+
+The following values for `mycfg`, `sycfg`, and `uycfg` are defined:
+
+[%autowidth,options=header,cols="^,^,<"]
+|===
+|Base Encoding|Features|Encoding
+| 0x0         | -      | Reserved (unspecified capability encoding)
+.2+|0x1       | 0x0    | <<app_cap_description>> capabilities, without <<section_zylevels1>> features
+              | 0x1   <| <<app_cap_description>> capabilities, with <<section_zylevels1>> features
```

Comment:
```suggestion
.2+|0x1       | 0x0    | <<app_cap_description, Default Encoding>> capabilities, without <<section_zylevels1>> features
              | 0x1   <| <<app_cap_description, Default Encoding>> capabilities, with <<section_zylevels1>> features
```

we need to come up with a name for them - and ideally set it on the heading of that appendix too - then this is good to go